libs/package.json

```diff
@@ -20,8 +20,8 @@
   "dependencies": {
     "@audius/hedgehog": "1.0.8",
     "@ethersproject/solidity": "5.0.5",
-    "@solana/spl-token": "^0.1.4",
-    "@solana/web3.js": "1.18.0",
+    "@solana/spl-token": "0.1.6",
```

Comment:
[this is same as current master version in package-lock, just pinning it](https://github.com/AudiusProject/audius-protocol/blob/master/libs/package-lock.json#L655)